test/Table_Tests/src/Database/Postgres_Spec.enso

```diff
@@ -21,7 +21,65 @@ import project.Database.Helpers.Name_Generator
 import project.Common_Table_Spec
 import project.Aggregate_Spec
 
-postgres_specific_spec connection pending =
+postgres_specific_spec connection db_name pending =
+    Test.group "[PostgreSQL] Schemas and Databases" <|
+        Test.specify "should be able to get current database and list databases" <|
+            connection.database . should_equal db_name
+            connection.databases.length . should_not_equal 0
+            connection.databases.contains db_name . should_be_true
+            Meta.is_same_object connection (connection.set_database db_name) . should_be_true
+
+        Test.specify "should be able to get current schema and list schemas" <|
+            connection.schema . should_equal "public"
+            connection.schemas.length . should_not_equal 0
+            connection.schemas.contains "public" . should_be_true
+            connection.schemas.contains "information_schema" . should_be_true
+            Meta.is_same_object connection (connection.set_schema "public") . should_be_true
+
+        Test.specify "should allow changing schema" <|
+            connection.set_schema "information_schema" . schema . should_equal "information_schema"
```

Comment:
Maybe this would be slightly clearer in what this test is doing?
```suggestion
            new_connection = connection.set_schema "information_schema"
            new_connection.schema . should_equal "information_schema"
```